x-pack/plugins/security_solution/public/common/components/event_details/cti_details/enrichment_accordion_group.tsx

```diff
@@ -98,7 +98,7 @@ const EnrichmentAccordion: React.FC<{
       key={accordionId}
       initialIsOpen={true}
       arrowDisplay="right"
-      buttonContent={<EnrichmentButtonContent field={field} provider={provider} value={value} />}
+      buttonContent={<EnrichmentButtonContent field={field} feedName={feedName} value={value} />}
```

Comment:
it looks like we are completely removing the `provider` string from the UI - I thought we wanted to display `provider` if there was no `feedName` for the purpose of identifying the source of a feed. I understand previously raised concerns around conflating information, but we also know that it's very unlikely for the users to have `threat.feed.name` fields populated at least in 8.0 (as `threat.feed.name` is not in the ECS specification). IMHO we can do something like `feedName={feedName ?? provider}` in the UI context